helm/openwhisk/values.yaml

```diff
@@ -197,7 +209,7 @@ invoker:
   containerFactory:
     dind: false
     useRunc: false
-    impl: "docker"
+    impl: "kubernetes"
```

Comment:
Let's leave the default here as docker due to poor cold start and log extraction performance of KubernetesContainerFactory. Change default downstream instead, or wait for #370 to get merged before enabling providers?

Reply:
Ok.. switching it back to docker